src/ddmd/dimport.d

```diff
@@ -155,9 +156,18 @@ extern (C++) final class Import : Dsymbol
                         else
                         {
                             // mod is a package.d, or a normal module which conflicts with the package name.
-                            assert(mod.isPackageFile == (p.isPkgMod == PKGmodule));
-                            if (mod.isPackageFile)
+                            pkgmod = mod.insertIntoPackageTree();
+                            if (auto mprev = pkgmod.isModule())
+                            {
+                                // https://issues.dlang.org/show_bug.cgi?id=14446
```

Comment:
This is a fix for 11847, not 14446. I'm confused.

Reply:
ping @MartinNowak

Reply:
That workaround/fix was there before, I just added the Bugzilla reference, as the comment wasn't too enlightening.